met-api/tests/unit/services/test_engagement.py

```diff
@@ -32,3 +33,16 @@ def test_create_engagement(session):  # pylint:disable=unused-argument
     assert fetched_engagement.get('description') == engagement_data.get('description')
     assert fetched_engagement.get('start_date')  # TODO address date format and assert
     assert fetched_engagement.get('end_date')
+
+
+def test_patch_engagement(session):  # pylint:disable=unused-argument
+    """Assert that an Org can be created."""
+    saved_engagament = factory_engagement_model()
+    engagement_edits = {
+        'id': saved_engagament.id,
+        'name': 'Updated engagement name'
+    }
+    assert engagement_edits['name'] != saved_engagament.name
```

Comment:
using faker to generate a random name can take away this assertion

Reply:
Done